packages/feature-toggles/src/resources/__tests__/StaticFeatureRepository.spec.ts

```diff
@@ -0,0 +1,31 @@
+import StaticFeatureRepository, { StaticFeatures } from '../StaticFeatureRepository';
+describe('StaticFeatureRepository', () => {
+    it('should respond with value true for available feature', async () => {
+        const features: StaticFeatures = {
+            'test-feature': {
+                value: true,
+                variation: 'T1',
+            },
+        };
+
+        const sut = new StaticFeatureRepository(features);
```

Comment:
What is "sut"?

Reply:
"System under test", generic test terminology.

Reply:
Basically it indicates the thing I am testing in this test.